File: security_corpus/go/sqli/vulnerable/sample.go
package main

import (
    "database/sql"
    "fmt"
    "os"
)

func getUser(db *sql.DB) {
    username := os.Args[1]
    query := fmt.Sprintf("SELECT * FROM users WHERE name = '%s'", username)
    db.Query(query)
}

File: security_corpus/python/sqli/regression/sample.py
import sqlite3


def get_user_regression():
    user_id = input('Enter user id: ')
    conn = sqlite3.connect("db.sqlite")
    cursor = conn.cursor()
    sql = "SELECT id, name FROM users WHERE id = " + str(user_id)
    cursor.execute(sql)
    return cursor.fetchone()
